tests/regression.sh

```diff
@@ -37,16 +46,20 @@ function cmp_output()
     printf "$cmd"
 
     if [[ $TRAVIS_OS_NAME == windows ]]; then
-        cmp_tool="diff --strip-trailing-cr -q"
+        cmp_tool="diff --strip-trailing-crq"
```

Comment:
Please keep `-q` separated, otherwise it confuses readers.

Reply:
This is a typo, I meant to remove -q.  diff with `-q` only says the files differ which is already implied in this context.

So all I really did here I just broke the tests for Windows. :)